assets/js/WfParameter.js

```diff
@@ -356,6 +359,24 @@ export default class WfParameter extends React.Component {
           </div>
         );
 
+      case 'url':
```

Comment:
I added this in error. I've since nixed it.

(We have a problem using copy/paste with single-line strings ... but that doesn't relate to this request)